pilot/pkg/bootstrap/server.go

```diff
@@ -70,6 +70,16 @@ const (
 	CopilotTimeout = 5 * time.Second
 	// FilepathWalkInterval dictates how often the file system is walked for config
 	FilepathWalkInterval = 100 * time.Millisecond
+
+	// TODO once topology labels are GA, stop using beta labels
+
+	// NodeRegionLabel is the well-known label for kubernetes node region
+	NodeRegionLabel = "failure-domain.beta.kubernetes.io/region"
+	// NodeZoneLabel is the well-known label for kubernetes node zone
+	NodeZoneLabel = "failure-domain.beta.kubernetes.io/zone"
+	// DefaultAvailabilityZone is default availability zone, used only if nodes
+	// do not have explicitly assigned.
+	DefaultAvailabilityZone = "localcluster"
```

Comment:
"default" ? 

Can you override it with 'os.Getenv("ISTIO_DEFAULT_AZ")' for example ? Also a comment that it'll apply to 
all nodes loaded by registries that don't set per-registry or per-node AZ labels.

Reply:
Who sets this environment variable and what to do if it is not set too?